### DISCUSSION TITLE
Pole na podwykonawce uslugi WZA

### DISCUSSION THREAD

**Discussion by d3stozaur** [allegro/allegro-api#8942] [Suggestions - resolved]
Metody Allegro One Box, czy Allegro One punkt moga byc realizowane przez DPD lub UPS i teraz dostając z endpointu https://api.allegro.pl/parcel-management/delivery-services metody dostaw, w tych metodach dostaw jedyna informacja o tym jaki kurier obsluguje dana przesylke jest tylko w nazwie co jest dosyć słabym rozwiązaniem i lepiej by było jakby zostało dodane dodatkowe pole z taką informacją. A jest to przydatne z tego powodu że jeśli generujemy przez zew. system przesyłki przez WZA albo bezpośrednio przez api jakiegoś kuriera i potem chcemy wygenerować z takich przesyłek raporty paczek dla kuriera.

**Comment by MaciejFrackowiak** [comment]
Informację zwracamy już w polu "packages.transportingInfo", o czym informowaliśmy w [newsie](https://developer.allegro.pl/news/wysylam-z-allegro-wprowadzilismy-zmiany-na-zasobach-do-zarzadzania-wysylka-przesylek-i-ich-odbiorem-przez-kuriera-oADdP41WVHA).
